code/game/objects/items/powerfist.dm

```diff
@@ -97,7 +97,7 @@
 		target.visible_message("<span class='danger'>[user]'s powerfist lets out a weak hiss as [user.p_they()] punch[user.p_es()] [target.name]!</span>", \
 			"<span class='userdanger'>[user]'s punch strikes with force!</span>")
 		return
-	target.apply_damage(force * fisto_setting, BRUTE, wound_bonus = -25*fisto_setting**2)
+	target.apply_damage(force * fisto_setting, BRUTE, wound_bonus = 10*fisto_setting-20)
```

Comment:
Is the order of operations the intended ones here in `10*fisto_setting-20`?
The lack of spaces and comments do not make this very clear.

Reply:
i realized powerfists are wack anyway and gave up trying quadratic formulas to make it reasonable with wounds so i just disabled wounds on the punch and will leave it to the throw effect to do this